Components/Alerts/alerts.css

```diff
@@ -0,0 +1,62 @@
+@import url("../../default.css");
+
+.error-alert{
+    background-color: var(--danger-color-shade);
+    color: var(--danger-color);
+    min-width: 25rem;
+    padding: 0.5rem;
+    border-radius: 0.4rem;
+}
+
+.success-alert{
+    background-color: var(--success-color-shade);
+    color: var(--success-color);
+    padding: 0.5rem;
+    min-width: 25rem;
+    border-radius: 0.4rem;
+}
+
+.warning-alert{
+    background-color: var(--warning-color-shade);
+    color: var(--dark-color);
+    padding: 0.5rem;
+    min-width: 25rem;
+    border-radius: 0.4rem;
+}
+
+.primary-alert{
+    background-color: var(--primary-color-shade);
+    color: var(--light-color);
+    padding: 0.5rem;
+    min-width: 25rem;
+    border-radius: 0.4rem;
+}
+
+@media only screen and (max-width: 768px) {
+    .error-alert{
+        background-color: var(--danger-color-shade);
+        color: var(--danger-color);
+        min-width: fit-content;
+        padding: 0.5rem;
+    }
```

Comment:
inside media query you don't need to write CSS of the class again
just mention changes that will work

eg:

.error-alert{
    background-color: var(--danger-color-shade);
    color: var(--danger-color);
    min-width: 25rem;
    padding: 0.5rem;
    border-radius: 0.4rem;
}
now In media query

@media only screen and (max-width: 768px) {
    .error-alert{
        min-width: fit-content;
    }
}

Reply:
Okay. I will do this.